site/assets/scss/_variables_project.scss

```diff
@@ -31,7 +32,30 @@ $danger: #ED6A5A !default;
 $white: #fff !default;
 $light: $mk-light;
 $medium: $mk-medium;
+$spacer: 0.5em;
 
+body, p {
+    font-family: 'Lora', serif !important;
+}
+
+
+h1,h2,h3,h4,h5 {
+    font-family: 'Open Sans', sans-serif !important;
+}
+
+.step {
+    margin-top: 2em;
+    font-family: 'Open Sans', sans-serif !important;
```

Comment:
not sure which one is the font for the Code Blocks
but how about we make the code block fonts, a Code Font...Like Currier or whatever looks Computer-Font

Reply:
I'm a bit confused - the code blocks are a code font right now: "Inconsolata". It seemed to be the one that worked the best with the other fonts (Lora & Open Sans), and decently narrow. Not your taste?

Reply:
Increased font size just a hair for clarity.